docs/content/latest/integrations/janusgraph.md

```diff
@@ -26,8 +26,13 @@ $ ycqlsh
 Connected to local cluster at 127.0.0.1:9042.
 [ycqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
 Use HELP for help.
+```
+
+```shell
```

Comment:
All ysqlsh and ycqlsh commands should be tagged as sql...

````suggestion
```sql
````

Reply:
Thanks. Changed.